lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
```

Comment:
updated default number of pins and only setup analog pins if passed in. 
Updated the analog pin read test as it now requires analog pins to be passed in.